content/guides/integrations/metaflow.md

```diff
@@ -34,6 +34,11 @@ For a more streamlined approach, you can generate an API key by going directly t
 
 To install the `wandb` library locally and log in:
 
+{{% alert %}}
+For wandb versions <= 0.19.8, install `fastcore<1.8.0` instead of `plum-dispatch`:
```

Comment:
Is there any way to avoid hard-coding all these versions, which will be difficult to maintain? Is there any reason not to have them install exactly 1.8.0, since it seems anything newer than that won't work?